lang/rust/rust-host.mk

```diff
@@ -0,0 +1,38 @@
+## SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 OpenWrt.org
```

Comment:
I think you are not allowed to put here OpenWrt.org as Copyright, since you are not a "official member" of the OpenWrt community. Please just remove it and also from all other files.

Reply:
Sure